tests/integration/test_dataplex_metadata.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from clouddq.integration.dataplex.clouddq_dataplex import CloudDqDataplexClient
+import logging
+
+logger = logging.getLogger(__name__)
+
+class TestMetadataIntegration:
+
+    @pytest.fixture
+    def test_dq_dataplex_client(self,
```

Comment:
could you move this client fixture to conftest and reuse it across test_dataplex_metadata.py and test_dataplex_task.py?